routing/router.go

```diff
@@ -366,8 +366,23 @@ func (a alice) then(h http.Handler) http.Handler {
 func buildHttpHandlerLogger(route Route, handlerFunction http.Handler) http.Handler {
 	c := alice{}
 	c = c.append(hlog.NewHandler(zerolog.With().Str("route", route.Name).Logger()))
-	c = c.append(hlog.HostHandler("Host", true))
-	c = c.append(hlog.RequestIDHandler("RequestID", "Request-Id"))
+	c = c.append(hlog.HostHandler("host", true))
+	c = c.append(hlog.RemoteAddrHandler("remoteAddr"))
```

Comment:
Why are we adding `remoteAddr` handler here as well? I know this info is useful but I'm not sure how much we should _always_ add. Maybe we can keep the additional info to a minimum unless you've set your log level to `debug` or `trace` and then we'll add more?

Reply:
I'm -1 on camel case. I think it's harder to read than either `RemoteAddr` or `remote-addr` but that's the nittiest of all so feel free to ignore.

Reply:
Now uses lowercase-with-dashes. Removed Host and Remote Addr for now - commented them out so we can resurrect easily if we want to in the future.

Here's what it looks like now.

```json
...

...
{
  "level": "trace",
  "route": "GraphNamespaces",
  "request-id": "d0hn9l19ks4s73apva20",
  "group": "graph",
  "appender": "idleNode",
  "ts": "2025-05-13T16:35:32Z",
  "msg": "Adding idle node for workload [prometheus] with labels [map[app.kubernetes.io/component:server app.kubernetes.io/instance:prometheus app.kubernetes.io/name:prometheus app.kubernetes.io/part-of:prometheus app.kubernetes.io/version:v3.1.0 helm.sh/chart:prometheus-26.1.0 sidecar.istio.io/inject:false]]"
}
```